build.fsx

```diff
@@ -82,6 +82,9 @@ pipeline "Build" {
     stage "Build" { run "dotnet build -c Release" }
     stage "Analyze" {
         run (fun _ -> async { System.IO.Directory.CreateDirectory(analysisReportsDir) |> ignore })
+        envVars
```

Comment:
I would put this closer to the tool invocation.
Can we do this right before the `<Exec Command`?

Reply:
Probably using https://learn.microsoft.com/en-us/visualstudio/msbuild/setenv-task?view=vs-2022